backend/application/syson-sysml-export/src/test/java/org/eclipse/syson/sysml/export/SysMLModelToTextSwitchTest.java

```diff
@@ -722,6 +725,25 @@ public void attributeUsageWithInvocationExpression() {
         AttributeUsageWithInvocationExpressionTestModel model = new AttributeUsageWithInvocationExpressionTestModel();
         this.assertTextualFormEquals("attribute attribute1 = fuel(front);", model.getAttributeUsage());
     }
+    
+    @Test
+    public void abstractAttributeUsageWithClassificationExpression() {
+        AttributeUsage attributeUsage = this.builder.createWithName(AttributeUsage.class, ATTRIBUTE1);
+        AttributeUsage type = this.builder.createWithName(AttributeUsage.class, "Syson");
```

Comment:
Why do you use an _AttributeUsage_ as _Type_? Shouldn't it be an _AttributeDefinition_ instead?

Reply:
done

Reply:
Why are you typing an _AttributeUsage_ with a _MetadataDefinition_. Unless I'm missing something here, the type should be an _AttributeDefinition_, as proposed in my first comment.